src/browser/components/buttons/index.jsx

```diff
@@ -290,13 +290,22 @@ const BaseCarouselButton = styled.button`
   width: 40px;
   min-width: 40px;
   -webkit-font-smoothing: antialiased;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none;
```

Comment:
`styled-components` generates vendor prefix, you don't have to.
See generated CSS below: 
![oskar4j 2017-04-11 at 13 15 06](https://cloud.githubusercontent.com/assets/570998/24906681/0a0a45c2-1eb9-11e7-8b0e-4aa477c21c87.png)